src/ml-service/packages/ml-api/met-forecast.py

```diff
@@ -0,0 +1,226 @@
+                            total_snow_amount =   forecast_timeseries[i].get('totalSnowAmount')
+                            prob_of_precipitation =  forecast_timeseries[i].get('probOfPrecipitation ')
+                            created_at  =  created_at
+                            airqo_channel_id = channel_id 
+                            elevation = point_elevation
+                            request_distance_point = distance_from_requested_point
+
+                            weather_predictions_tuple = (location_name, location_latitude, location_longitude, time,
+                                screen_temperature, screen_dewpoint_temperature, feels_like_temperature, wind_speed_10m,
+                                wind_direction_from_10m, wind_gust_speed_10m, visibility, screen_relative_humidity,
+                                 mean_sea_level_pressure, uvIndex, significant_weather_code, precipitation_rate,
+                                 total_precipitation_amount, total_snow_amount, prob_of_precipitation, created_at,
+                                  airqo_channel_id, elevation, request_distance_point)
+
+
+                            weather_predictions.append(weather_predictions_tuple)
+
+
+            results = save_weather_forecasts(weather_predictions)
+
+
+        else:
+            return "Failed to get forecasts for specified coordinates", status_code
+
+
+
+def get_hourly_met_forecasts():
+    """
+        gets hourly spot wether forecasts from met 
+        office(https://metoffice.apiconnect.ibmcloud.com/metoffice/production/)
+        saves them to big query table.
+    """
+    weather_predictions=[]
+    specified_locations = get_all_static_channels()
+    #specified_locationsx = specified_locations[0:2]
+    if specified_locations:
+        for i in range(0, len(specified_locations)):
+            channel_id = specified_locations[i].get('channel_id')
+            latitude = specified_locations[i].get('latitude')
+            longitude = specified_locations[i].get('longitude')
+
+            forecast_results, status_code = get_location_hourly_weather_forecasts(latitude, longitude)
+
+            if status_code == 200 :
+                features_geometry = forecast_results['features'][0]['geometry']
+                features_properties = forecast_results['features'][0]['properties']
+                forecast_timeseries = features_properties['timeSeries']
+                location = features_properties['location'].get('name')
+                location_lat = features_geometry.get('coordinates')[1]    
+                location_long = features_geometry.get('coordinates')[0]
+                point_elevation = features_geometry.get('coordinates')[2]
+                distance_from_requested_point = features_properties['requestPointDistance']
+                created_at  =  datetime.now()
+
+                if forecast_timeseries:
+                    for i in range(0, len(forecast_timeseries)):
```

Comment:
same here, we could explore `destructuring `when assigning values to variables from the same "source:"